src/pycrunch/shoji.py

```diff
@@ -151,7 +151,7 @@ def create(self, entity=None, progress_tracker=None):
         elif isinstance(entity, dict) and not isinstance(entity, elements.Document):
             entity = Entity(self.session, **entity)
 
-        response = self.post(data=entity.json)
+        response = self.post(data=json.dumps(entity))
```

Comment:
Can we add: `indent=None, seperators=(',', ':')` to the `json.dumps` call please?

This way we get the most compact representation possible. No whitespace is necessary after the `:`, and large dictionaries will have a lot of them.

Reply:
Done in a41f0fb05ada261abdc2d7d56306a4ff84d1478b.